scripts/core/EXT_Exp_CounterBasedEvents.rst

```diff
@@ -0,0 +1,56 @@
+<%
+import re
+from templates import helper as th
+%><%
+    OneApi=tags['$OneApi']
+    x=tags['$x']
+    X=x.upper()
+%>
+:orphan:
+
+.. _ZE_experimental_event_pool_counter_based:
+
+=========================
+ Counter-Based Event Pools Extension
+=========================
+
+API
+----
+ 
+* Enumerations
+
+    * ${x}_event_pool_counter_based_exp_flags_t
+    * ${x}_event_pool_counter_based_exp_version_t
+
+* Structures
+
+    * ${x}_event_pool_counter_based_exp_desc_t
```

Comment:
is there a CounterBasedEvents.yml file where you have expanded on this descriptor? not seeing that as part of this PR (or am I not looking at right view!?)

Reply:
@aravindksg : files were not added. Please see now.